src/components/templates/ErrorPages/Maintenance/Maintenance.tsx

```diff
@@ -34,17 +29,17 @@ const Maintenance: React.FC<FiveHundredPageTemplateProps> = ({ helpLine = HelpLi
         </FlexCol>
       </FlexRow>
       <Heading as="h1" size={width <= breakpoints.desktop ? 'h4' : 'h3'} align="center" className="mb-4">
-        We’ll be back soon!
+        We'll be back soon!
       </Heading>
       <Text as="p" size={width <= breakpoints.desktop ? 'body' : 'lead'} align="center" className="mb-7">
-        Sorry for the inconvenience but we’re doing some essential work on our website right now.
+        Sorry for the inconvenience but we're doing some essential work on our website right now.
       </Text>
       <Text as="p" size="small" align="center" className="mb-7">
-        Please come back later or give us a call on <Link href={`tel:${helpLine}`}>{helpLine}</Link> and we’ll help you
-        out.
+        Please come back later or give us a call on{' '}
+        <Link href={`tel:${details[helpLine].telephone.label}`}>{details[helpLine].telephone.label}</Link> and we'll
```

Comment:
[Amend] Noticed that this needs changing to `tel`
```suggestion
        <Link href={`tel:${details[helpLine].telephone.tel}`}>{details[helpLine].telephone.label}</Link> and we'll
```